### PR TITLE
Added configurations for brazilian real (BRL) currency

### PR DESCRIPTION
**Pull request by ronflima**
Added formatting for BRL currency according to code documentation.

### CODE REVIEW

File: subscriptions/conf.py [hunk 1]
```diff
@@ -51,6 +51,7 @@ class Currency():
             3: The sign should immediately precede the value.
             4: The sign should immediately follow the value.
     """
+
```
Comment (studybuffalo) [inline subscriptions/conf.py:54]: Remove new line

File: subscriptions/conf.py [hunk 2]
```diff
@@ -166,6 +167,7 @@ def format_currency(self, value, international=False):
 
         return formatted_currency
 
+
```
Comment (studybuffalo) [inline subscriptions/conf.py:170]: Remove new line

File: subscriptions/conf.py [hunk 3]
```diff
@@ -177,6 +179,7 @@ def string_to_module_and_class(string):
         'class': component_class,
     }
 
+
```
Comment (studybuffalo) [inline subscriptions/conf.py:182]: Remove new line

**Comment by ronflima** [comment]
> 
> 
> Thanks for the contribution! I noticed a few extra new lines were added to the file that I have flagged for removal just to keep the styling consistent. If you have a moment to take those out it would be appreciated. Afterwards this is good for approval and I can get a new version out.

I´ll do that. I´m using autopep8 to force code styling consistent with python standards. I´ll check if this was forced by it.

**Comment by ronflima** [comment]
Just made requested changes. I have added, also, .pep8 configuration to avoid autopep8 break stablished code formatting by adding 2 lines at the end of each function or class (E305).  Also, added a file to .gitignore to avoid saving an emacs configuration file to the repo.

**Comment by studybuffalo** [comment]
Ah thanks for the extra info. I must have missed/forgot about 2 newlines between functions per PEP8. I try to keep this package in line with PEP wherever possible and will make sure to fix that.

To keep the git history clear/consistent, would you be able to pull out the .pep8 file and I will do a new PR in the next week or two to bring the package into alignment with this PEP8 standard? Sorry to be a pain and I really appreciate you bringing it to my attention!

**Comment by ronflima** [comment]
Don´t worry. I have added the .pep8 only to force autopep8 to follow your standards. Auto-formatters are great to ensure code style compliance. I´ll take it out from the commit. BTW, if you want, I can format the entire code to pep8 standards. It is a matter of running a script to automate this task. But I will add it in another PR. Let me know if you want it.

**Comment by studybuffalo** [comment]
Thanks! It is 100% your call if you'd like to update the formatting to align. I guess the one caveat I will add: there are probably some deviations made that align more with Django styles than Python. I tried to stick to Python standards wherever possible, but there may have been some areas where I felt Django was more readable. I may be a bit nitpicky around those areas, but hopefully it doesn't come up too often if I stuck to the style guide as intended!

**Comment by studybuffalo** [comment]
Also, I will try and get a new version out this weekend to have these changes added to the package. Today is "dependency update" day, so just want to get those all done before putting out a new version in case there are any dependency conflicts I need to resolve (unlikely, but you never know).

**Pull request merged**